site/src/components/ui/command.tsx

```diff
@@ -0,0 +1,151 @@
+import type { DialogProps } from "@radix-ui/react-dialog";
+import { Command as CommandPrimitive } from "cmdk";
+import { Search } from "lucide-react";
+import * as React from "react";
+
+import { Dialog, DialogContent } from "components/ui/dialog";
+import { cn } from "utils/cn";
+
+const Command = React.forwardRef<
+	React.ElementRef<typeof CommandPrimitive>,
+	React.ComponentPropsWithoutRef<typeof CommandPrimitive>
+>(({ className, ...props }, ref) => (
+	<CommandPrimitive
+		ref={ref}
+		className={cn(
+			"flex h-full w-full flex-col overflow-hidden rounded-md bg-popover text-content-primary",
+			className,
+		)}
+		{...props}
+	/>
+));
+Command.displayName = CommandPrimitive.displayName;
```

Comment:
React should infer good display names for this without needing to set it manually. if not, it's a tooling problem.

Reply:
Another shadcn thing, but to confirm, is the code in a state where it is already inferring display names?